tests/controllers/management-tests/contract-negotiations.test.ts

```diff
@@ -140,7 +140,6 @@ describe("ContractNegotiationController", () => {
     it("terminate the requested target negotiation", async () => {
       const { idResponse } = await createContractNegotiation(provider, consumer);
       const negotiationId = idResponse.id;
-      await waitForNegotiationState(consumer, negotiationId, "FINALIZED");
```

Comment:
why this line was deleted?

Reply:
This must have been a mistake, I apologize.